openprescribing/matrixstore/row_grouper.py

```diff
@@ -121,6 +118,20 @@ def sum_one_group(self, matrix, group_id):
 def has_type_matrix(value):
     """
     Return whether `value` is a numpy array or a numpy matrix
+
```

Comment:
I've found, twice, when reading this code, that I parse "has_type_matrix" as "has a type matrix", and then I wonder what a "type matrix" is.

What about renaming to "is_matrix"?